go/exorelay/exorelay_test.go

```diff
@@ -73,6 +74,7 @@ func FeatureContext(s *godog.Suite) {
 	})
 
 	s.Step(`^ExoRelay connects to Exocom$`, func() error {
+		time.Sleep(time.Millisecond * 1)
```

Comment:
I think it's not in the scope of this PR but it's something that we should be doing